### PR TITLE
feat: React migration - PR list CY-6556

### PR DESCRIPTION
**Pull request by claudiacarpinteiro**
Updates the page Quality Pull Requests page to take into account the changes introduced while migrating the Pull Requests list page to React.

### :eyes: Live preview
https://feat-update-quality-pull-requests-c--docs-codacy.netlify.app/repositories/pull-requests/

### :construction: To do
-   [x] Update screenshot with final design of the page
-   [x] Wait for feature release

### CODE REVIEW

File: docs/repositories/pull-requests.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ file_name: "pull-requests"
...
 # Quality Pull Requests page
 
-The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the number of new and fixed issues for each pull request. This allows you to monitor the code quality of the work in progress in your repository.
+The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the code quality metrics for each pull request. This allows you to monitor the code quality of the work in progress in your repository.
 
-By default, the page lists open pull requests but you can use the drop-down list at the top of the page to display the closed pull requests.
+By default, the page lists open pull requests but you can click the **Closed** tab at the top of the list to display the closed pull requests.
 
-![Pull Requests page](images/pull-requests.png)
+![Pull Requests page](images/pull-requests.png)<!--TODO CY-6556 Update screenshot. It might be tricky to update this screenshot only, because all the remaining screenshots for the Pull Request page are related with one-->
```
Comment (nicklem) [inline docs/repositories/pull-requests.md:12]: Let's sync on this during the daily.

File: docs/repositories/pull-requests.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ file_name: "pull-requests"
 
 # Quality Pull Requests page
 
-The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the number of new and fixed issues for each pull request. This allows you to monitor the code quality of the work in progress in your repository.
+The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the code quality metrics for each pull request. This allows you to monitor the code quality of the work in progress in your repository.
 
-By default, the page lists open pull requests but you can use the drop-down list at the top of the page to display the closed pull requests.
+By default, the page lists open pull requests but you can click the **Closed** tab at the top of the list to display the closed pull requests.
```
Comment (nicklem) [inline docs/repositories/pull-requests.md:10]: A couple of suggestions:
- split this into two sentences 
- invert the second sentence to front-load the result with the structure `To [result], [action].`

```suggestion
By default, the page lists open pull requests. To display closed pull requests, click the **Closed** tab above the list.
```

Comment (nicklem) [reply docs/repositories/pull-requests.md:10]: Let's go with 2 for now 👍

File: docs/repositories/pull-requests.md (hunk 1 above)
Comment (prcr) [reply docs/repositories/pull-requests.md:12]: This is the open source repository that I used to create the original screenshot:

https://app.codacy.com/gh/pyccel/pyccel/pullRequests

You should be able to grab an updated screenshot from that link.

**Review by inesgomas — APPROVED** [review]

**Pull request merged**